tests/test_routes.py

```diff
@@ -366,24 +367,97 @@ def test_update_inventory(self):
         """Update an existing record in Inventory"""
         # create a record in Inventory
         inventory = InventoryFactory()
+        inventory.condition = Condition.NEW
         resp = self.app.post(
             BASE_URL, json=inventory.serialize(), content_type="application/json"
         )
         self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
-        # update the record
-        new_inventory = resp.get_json()
-        logging.debug(new_inventory)
-        new_inventory["quantity"] = 999
-        new_inventory["restock_level"] = 99
+        # update quantity and restock_level
+        params = {}
+        params[QUANTITY] = 1000
+        params[RESTOCK_LEVEL] = 400
         resp = self.app.put("/inventory/{}/condition/{}".format(
-            new_inventory["product_id"], new_inventory["condition"]),
-            json=new_inventory,
+            inventory.product_id, inventory.condition.name),
+            json=params,
+            content_type="application/json",
+        )
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+        updated_inventory = resp.get_json()
+        self.assertEqual(updated_inventory[QUANTITY], 1000)
+        self.assertEqual(updated_inventory[RESTOCK_LEVEL], 400)
+        # update quantity by added_amount
+        params = {}
```

Comment:
Constantly modifying the variable `params` is not recommended. Maybe we can directly do something like `params={QUANTITY: 1000, RESTOCK_LEVEL: 400}`.
What do you think?